guides/serialization_and_saving.py

```diff
@@ -17,18 +16,19 @@
 - A set of weights values (the "state of the model").
 - An optimizer (defined by compiling the model).
 - A set of losses and metrics (defined by compiling the model or calling
-`add_loss()`).
+`add_loss()` or `add_metric()`).
```

Comment:
All references to add_metric should be scrubbed

Reply:
Removed those references, thanks!